src/style.css

```diff
@@ -1,3 +1,92 @@
+/* CSS Variable for the blue color */
+:root {
+  --blue-color: #00ACDC;
+}
+
+/* Reset browser margins */
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
+  background-color: #fff;
+  color: #000;
+}
+
+/* Header */
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 40px;
```

Comment:
Hey Gabriel. Look, your header shouldn't have any vertical padding, it makes the navbar smaller than it should be.

Also, the horizontal padding on yout header is meant to be 50px, following the mockup, instead of 40px.

Reply:
It seems you're trying to use the vertical padding to ensure the height of the header. Instead, you should define the height of the links in your nav